packages/x64dbg.dbgchild.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,61 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+    $toolSrcDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
```

Comment:
[nitpick] For other similar packages (for example [idr.vm](https://github.com/mandiant/VM-Packages/blob/main/packages/idr.vm/tools/chocolateyinstall.ps1#L15)), we create a temp directory for cases like this. @vm-packages any preference on how we should do this in general? I think I tend to prefer a temp directory as we don't plan to keep it as the tool directory.

Reply:
I changed it to be similar to the `idr.vm` example you linked